app/gen-server/lib/homedb/GroupsManager.ts

```diff
@@ -0,0 +1,275 @@
+import { Group } from "app/gen-server/entity/Group";
+import { GroupDescriptor, NonGuestGroup, Resource } from "app/gen-server/lib/homedb/Interfaces";
+import * as roles from 'app/common/roles';
+import { Permissions } from 'app/gen-server/lib/Permissions';
+import { User } from "app/gen-server/entity/User";
+import { Organization } from "app/gen-server/entity/Organization";
+import { Workspace } from "app/gen-server/entity/Workspace";
+import { Document } from "app/gen-server/entity/Document";
+import { AclRule } from "app/gen-server/entity/AclRule";
```

Comment:
Ideally imports are ordered alphabetically, and use of quotes are consistent. Sorry we don't have automatic linting for this.

Reply:
@paulfitz I am never really sure about how to sort the imports. How should we sort this line: 
`import { GroupDescriptor, NonGuestGroup, Resource } from "app/gen-server/lib/homedb/Interfaces";` 

Should we consider the imported object (`GroupDescriptor`) or the path (`"app/gen-server/lib/homedb/Interfaces"`) for sorting?

Reply:
The path